docs/product/dev-toolbar/setup.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: "Set Up"
+sidebar_order: 10
+description: "Get started with Sentry's Dev Toolbar, bringing critical Sentry insights and tools into your web app to help your team troubleshoot more effectively.."
+---
+
+<Alert level="info">
+  The Dev Toolbar is currently in **beta**. Beta features are still in progress and may have bugs. Please reach out on
+  [GitHub](https://github.com/getsentry/sentry-toolbar/issues) if you have any feedback or concerns.
+</Alert>
+
+## Set Up Your Web App
+
+<Alert title="Note" level="warning">
+    Enabling performance is a prerequisite for having page-aware panels, as it allows us to collect page-specific issues and feedback through transactions.
```

Comment:
We refer to it as 'tracing' instead of performance. But, what we really mean here is that a transaction must exist and be bound to the scope? I think given the description it makes sense, I'd just rename it to tracing, and link to the docs. 

```suggestion
    [Enabling tracing](/platforms/javascript/tracing/) is a prerequisite for having page-aware panels, as it allows us to collect page-specific issues and feedback through transactions.
```